docs/css/custom.css

```
@@ -28,27 +54,27 @@
 
 .md-header__button.md-logo img{
   display: block;
-  width:90px;
+  width: 110px;
+  height: 85px;
   fill: currentColor;
 }
 
-
 /* ########## Bootstrap Button CSS ########## */
 
 /* ########## Bootstrap Button CSS ########## */
 .btn-success {
-  color: #fff !important;
-  background-color: #009485 !important;
-  border-color: #009485 !important;
+  color: #212529 !important;
+  background-color: #ffbc42 !important;
+  border-color: #ffbc42 !important;
  }
  .btn-success:hover, .btn-outline-success:hover {
-  color: #fff !important;
-  background-color: #26a699 !important;
-  border-color: #26a699 !important;
+  color: #212529 !important;
+  background-color: #ffbc42 !important;
+  border-color: #ffbc42 !important;
  }
  .btn-outline-success {
-  color: #26a699 !important;
-  border-color: #26a699 !important;
+  color: #212529 !important;
```

I think here should be `#ffbc42`, since outlined buttons use border color as color too on material
```suggestion
  color: #ffbc42 !important;
```